sdcm/tester.py

```diff
@@ -2066,6 +2067,29 @@ def run_fullscan_thread(self, ks_cf='random', interval=1, duration=None):
             password=sla_role_password,
         ).start()
 
+    def run_tombstone_gc_verification_thread(self, duration=None, interval=600, propagation_delay_in_seconds=3600, **kwargs):
+        """Run a thread of tombstones gc verification.
+
+        1. Search for repair history.
+        2. Go over node sstables, created after last repair.
+        3. Run an sstable dump to get sstables data.
+        4. Go over the output, searching for tombstones.
+        5. If a tombstone creation time (+propagation-delay) is earlier than last repair then ==>
+            raise a tombstone-gc-verification error.
+
+        Keyword Arguments:
+            interval {number} -- interval between requests in seconds (default: {600})
+            duration {int} -- duration of running thread in min (default: {None})
+        """
+        TombstoneGcVerificationThread(
+            db_cluster=self.db_cluster,
+            termination_event=self.db_cluster.nemesis_termination_event,
+            duration=self.get_duration(duration),
+            interval=interval,
+            propagation_delay_in_seconds=propagation_delay_in_seconds,
```

Comment:
you have `propagation_delay_in_seconds` set in a `.yaml` file, but you never look into it